File: src/main/java/Psychic/Core/Mana/Join/Join.java
package Psychic.Core.Mana.Join;

import Psychic.Core.Main.Depend.Psychic;
import Psychic.Core.Mana.Manager.ManaManager;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class Join implements Listener {
    @EventHandler
    public void onJoin(PlayerJoinEvent event) {
        ManaManager.removeAllBars();
        ManaManager.initAll(Psychic.getInstance());
    }
}

File: src/main/java/Psychic/Core/Mana/Join/Join.kt
package Psychic.Core.Mana.Join

import Psychic.Core.Main.Depend.Psychic
import Psychic.Core.Mana.Manager.ManaManager
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

class Join: Listener {
    @EventHandler
    fun onJoin(event: PlayerJoinEvent?) {
        ManaManager.removeAllBars()
        ManaManager.initAll(Psychic.getInstance())
    }
}